playbooks/ansible-cloud/k8s/downstream_sanity.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: all
+  tasks:
+    - name: Ensure controller directory exists
+      file:
+        path: "{{ dest }}"
+        state: directory
+
+    - name: Clone redhat.openshift collection
```

Comment:
You can just use pass this project in the `required-projects` list. This way Zuul will pull it for you.